minecraft_servers/griefergames/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+    "server_name": "GrieferGames",
+    "nice_name": "GrieferGames Netzwerk",
+    "direct_ip": "griefergames.net",
+    "social": {
+      "web": "https://griefergames.net/",
+      "web_shop": "https://store.griefergames.net/",
+      "web_support": "https://bit.ly/griefergames-support",
+      "twitter": "GrieferGamesNW",
+      "facebook": "GrieferGamesNW",
+      "instagram": "griefergamesnw",
+      "youtube": "https://www.youtube.com/user/AbgegrieftHD",
```

Comment:
Since it's not the network's YouTube account as such, I would remove it.

Reply:
I removed it